CRM/Activity/Import/Parser/Activity.php

```diff
@@ -436,4 +404,62 @@ protected function deprecated_activity_formatted_param(&$params, &$values, $crea
     return NULL;
   }
 
+  /**
+   *
+   * Get the value for the given field from the row of values.
+   *
+   * @param array $row
+   * @param string $fieldName
+   *
+   * @return null|string
+   */
+  protected function getFieldValue(array $row, string $fieldName) {
+    return $row[$this->getFieldIndex($fieldName)] ?? NULL;
```

Comment:
This needs to check if `$this->getFieldIndex($fieldName)` returns false, because suppose I map type id but not type label, then the false gets cast to element 0, and then on line 454 it's wrong and the `if` just below that does the wrong thing.

Reply:
@demeritcowboy I've updated it